draft-ietf-webpush-encryption.md

```diff
@@ -279,32 +279,25 @@ application server MUST set the `rs` parameter in the `aes128gcm` content coding
 header to a size that is greater than the length of the plaintext, plus any
 padding (which is at least 2 octets).
 
-A push message MUST include a zero length `keyid` parameter in the content
-coding header.
+A push message MUST include the application server ECDH public key in the
+`keyid` parameter of the encrypted content coding header.
```

Comment:
Well, extensibility is hazardous without knowing what you need it for. At this point, I'd say that it is best achieved by using an entirely different scheme.

Losing extensibility is one of the costs of this optimization.  If you really think we need extensibility, then this PR is a bad idea and we should go back to using Crypto-Key.